recipe/meta.yaml

```diff
@@ -30,11 +30,13 @@ requirements:
   host:
     - r-base
     - r-sp >=1.1_0
-    - libgdal
+    - gdal >=1.11.4,<=2.5.0
+    - proj4 >=4.8.0,<6
```

Comment:
You should not pin it like this b/c you may never get a solvable solution. Let the central pinning handle it.